### PR TITLE
A22AB2-6

### PR DESCRIPTION
**Pull request by ndwilliamson123**
Hey everyone. I connected the Warehouses List component to our backend and deleted the temp data on the front end.

### CODE REVIEW

File: src/components/WarehousesList/WarehousesList.js [hunk 1]
```diff
@@ -1,37 +1,56 @@
...
+    componentDidMount() {
+        axios
+            .get(`http://localhost:8080/warehouses`)
+            .then((response) => {
+                this.setState({
+                    warehouses: response.data,
+                });
+            })
+            .catch((error) => {
+                console.log(error);
```
Comment (ankurbag) [inline src/components/WarehousesList/WarehousesList.js:26]: Just a recommendation : If we have some time, if we can also alert the error on the UI will be great.

File: src/assets/TEMP_DATA/warehouses.json [hunk 2]
```diff
@@ -1,106 +0,0 @@
-[
```
Comment (ankurbag) [inline src/assets/TEMP_DATA/warehouses.json:1]: We can delete `TEMP_DATA` folder as well, if it is not required

Comment (ndwilliamson123) [reply src/assets/TEMP_DATA/warehouses.json:1]: That folder still has the Inventory data. Only the Warehouses backend is working right now.

File: src/components/WarehousesList/WarehousesList.js (hunk 1 above)
Comment (ndwilliamson123) [reply src/components/WarehousesList/WarehousesList.js:26]: Okay! I can add the Window alert() at least

Comment (ankurbag) [reply src/components/WarehousesList/WarehousesList.js:26]: Perfect

File: src/assets/TEMP_DATA/warehouses.json (hunk 2 above)
Comment (ankurbag) [reply src/assets/TEMP_DATA/warehouses.json:1]: Okay! Thanks for clearing that up.

**Review by ankurbag — APPROVED** [review]

**Comment by ankurbag** [comment]
Great wrk!!!

**Pull request merged**